### PR TITLE
feat(Env): Add boolean parser & preset

### PR DESCRIPTION
**Pull request by runk**

### CODE REVIEW

**Review by etaoins — APPROVED** [review]

File: src/env/parsers.test.ts [hunk 1]
```diff
@@ -53,3 +53,24 @@ describe('oneOf', () => {
...
+  const sadCases = [
+    [''],
+    ['false'],
+    ['no'],
+    ['off'],
+    ['0'],
+    ['False'],
+  ] as const;
+
+  it.each(sadCases)('parses %s', (input) =>
```
Comment (72636c) [inline src/env/parsers.test.ts:73]:
```suggestion
  const sadCases = ['', 'false', 'no', 'off', '0', 'False'];

  it.each(sadCases)('parses %p', (input) =>
```

File: src/env/parsers.test.ts [hunk 2]
```diff
@@ -53,3 +53,24 @@ describe('oneOf', () => {
       `"process.env.ENVIRONMENT is not a supported choice: 'dev'. Expected one of: ['local', 'prod']"`,
     ));
 });
+
+describe('boolean', () => {
+  const happyCases = [['1'], ['true'], ['yes'], ['on'], ['enabled']] as const;
+
+  it.each(happyCases)('parses %s', (input) =>
```
Comment (72636c) [inline src/env/parsers.test.ts:60]: I think Jest has a cheat here

```suggestion
  const happyCases = ['1', 'true', 'yes', 'on', 'enabled'];

  it.each(happyCases)('parses %p', (input) =>
```

File: src/env/parsers.ts [hunk 3]
```diff
@@ -30,3 +30,8 @@ export const oneOf = <T>(choices: readonly T[]) => {
     return input;
   };
 };
+
+const falseys = ['false', 'off', 'no', '0'];
```
Comment (72636c) [inline src/env/parsers.ts:34]: Do we want to support `n` for full YAML compatibility 😅

**Review by 72636c — APPROVED** [review]
Awesome, thank you!

**Comment by runk** [comment]
@72636c want to have another look?

**Pull request merged**